Description: Switch static lib to shared.
 <long description that can span multiple lines, optional>
Author: <name and email of author, optional>
Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-<Vendor>: <URL to the vendor bug report if any, optional>
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
Reviewed-by: <name and email of a reviewer, optional>
Last-Update: <YYYY-MM-DD, last update of the meta-information, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gatb-core/src/CMakeLists.txt
+++ b/gatb-core/src/CMakeLists.txt
@@ -83,7 +83,7 @@
 
 include_directories (${gatb-core-includes})
 
-add_library    (gatbcore-static  STATIC  ${ComponentFiles} )
+add_library    (gatbcore-static  SHARED  ${ComponentFiles} )
 #add_library   (gatbcore-dynamic SHARED  ${ComponentFiles} )
 
 target_link_libraries  (gatbcore-static hdf5 z pthread)
